<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>639</width>
    <height>485</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0">
     <widget class="QTabWidget" name="tabWidget">
      <property name="currentIndex">
       <number>0</number>
      </property>
      <widget class="QWidget" name="tab">
       <attribute name="title">
        <string>Products</string>
       </attribute>
       <layout class="QGridLayout" name="gridLayout_2">
        <item row="0" column="0">
         <widget class="QMdiArea" name="ProductMdiArea"/>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="tab_2">
       <attribute name="title">
        <string>Users</string>
       </attribute>
       <layout class="QGridLayout" name="gridLayout_3">
        <item row="0" column="0">
         <widget class="QMdiArea" name="userMdiArea"/>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="tab_3">
       <attribute name="title">
        <string>Orders</string>
       </attribute>
       <layout class="QGridLayout" name="gridLayout_4">
        <item row="0" column="0">
         <widget class="QMdiArea" name="ordersMdiArea"/>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="tab_4">
       <attribute name="title">
        <string>Addresses</string>
       </attribute>
       <layout class="QGridLayout" name="gridLayout_5">
        <item row="0" column="0">
         <widget class="QMdiArea" name="addressMdiArea"/>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>639</width>
     <height>22</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <widget class="QMenu" name="menuDatabase">
     <property name="title">
      <string>Database</string>
     </property>
     <addaction name="actionAdd_Products"/>
     <addaction name="actionAdd_Users"/>
    </widget>
    <widget class="QMenu" name="menuOrders">
     <property name="title">
      <string>Orders</string>
     </property>
     <addaction name="actionCheck_Orders_2"/>
     <addaction name="actionCreate_Order_3"/>
    </widget>
    <addaction name="separator"/>
    <addaction name="separator"/>
    <addaction name="menuDatabase"/>
    <addaction name="actionQuit"/>
    <addaction name="menuOrders"/>
    <addaction name="actionotro"/>
   </widget>
   <widget class="QMenu" name="menuHelp">
    <property name="title">
     <string>Help</string>
    </property>
    <addaction name="actionAbout"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuHelp"/>
  </widget>
  <action name="actionQuit">
   <property name="text">
    <string>Quit</string>
   </property>
  </action>
  <action name="actionAbout">
   <property name="text">
    <string>About</string>
   </property>
  </action>
  <action name="actionAdd_Products">
   <property name="text">
    <string>Add Products</string>
   </property>
  </action>
  <action name="actionAdd_Users">
   <property name="text">
    <string>Add Users</string>
   </property>
  </action>
  <action name="actionCreate_Order">
   <property name="text">
    <string>Create Order</string>
   </property>
  </action>
  <action name="actionAdd_address">
   <property name="text">
    <string>Add address</string>
   </property>
  </action>
  <action name="actionCheck_Orders">
   <property name="text">
    <string>Check Orders</string>
   </property>
  </action>
  <action name="actionCheck_Orders_2">
   <property name="text">
    <string>Check Orders</string>
   </property>
  </action>
  <action name="actionCreate_Order_3">
   <property name="text">
    <string>Create Order</string>
   </property>
  </action>
  <action name="actionSearch_Product">
   <property name="text">
    <string>Search Product</string>
   </property>
  </action>
  <action name="actionDatabase_2">
   <property name="text">
    <string>Database</string>
   </property>
  </action>
  <action name="actionotro">
   <property name="text">
    <string>otro</string>
   </property>
  </action>
  <action name="actionList_Users">
   <property name="text">
    <string>List Users</string>
   </property>
  </action>
  <action name="actionList_Products">
   <property name="text">
    <string>List Products</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
